packages/esm-form-entry-app/package.json

```diff
@@ -47,6 +47,8 @@
     "@angular/router": "~12.2.16",
     "@carbon/styles": "~1.14.0",
     "@ng-select/ng-select": "^6.1.0",
+    "@ngx-translate/core": "^13.0.0",
+    "@ngx-translate/http-loader": "^6.0.0",
```

Comment:
I don't think we need this at all...

Reply:
@ibacher do you mean only the `http-loader` or even the `core` ?